### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN040.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#87] [General]
# https://blog.3place.cn/3place/lmyn/LMYN040.html

https://blog.3place.cn/3place/lmyn/LMYN040.html

**Comment by picklecai** [comment]
方位象，这个设定有意思。象力气大，可背负大地。

**Comment by picklecai** [comment]
在那里他们看到伽毗罗，
他是永恒婆苏帝婆所变；
还有那一匹丢掉了的马，
走在离神仙不远的后边。 （1.39.24）

因陀罗偷的祭马。
因陀罗和这位婆苏帝婆/伽毗罗，是啥关系呢？
